app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -100,7 +100,8 @@ import org.simple.clinic.util.room.UuidRoomTypeConverter
     DeletedReason.RoomTypeConverter::class,
     MedicineFrequency.RoomTypeConverter::class,
     TeleconsultationType.RoomTypeConverter::class,
-    RoomTypeConverter::class
+    RoomTypeConverter::class,
+    User.CapabilityStatus.TypeAdapter::class
```

Comment:
~~You have to pass the `User. CapabilityStatus.RoomTypeConverter::class` here~~ NVM it has been done in next commit